### PR TITLE
Add reuse pvt key jwt checkbox to protocol tab

### PR DESCRIPTION
**Pull request by mpmadhavig**
### Purpose

$subject

Related Issues:
- https://github.com/wso2/product-is/issues/20546

### CODE REVIEW

File: modules/i18n/src/translations/en-US/portals/applications.ts [hunk 1]
```diff
@@ -1629,6 +1629,10 @@ export const applications: ApplicationsNS = {
                             label: "Client authentication method",
                             placeholder: "Select method"
                         },
+                        reusePvtKeyJWT: {
+                            hint: "By default private key JWT is not allowed to reuse. But if you have a requirment to reuse it. Please click on the checkbox.",
```
Comment (pavinduLakshan) [inline modules/i18n/src/translations/en-US/portals/applications.ts:1633]: It'd be better to get this text reviewed with the doc and ux teams. 

cc: @Achintha444 @savindi7 @himeshsiriwardana 

The following is just a suggestion.

```suggestion
                            hint: "Whether the private key JWT can be used multiple times. By default, private key JWTs are not allowed to be reused.",
```

File: features/admin.applications.v1/components/forms/inbound-oidc-form.tsx [hunk 2]
```diff
@@ -1930,6 +1932,39 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
...
+                        { selectedAuthMethod === PRIVATE_KEY_JWT &&
+                            (
+                                <Grid.Row columns={ 1 }>
+                                    <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                                        <Field
+                                            ref={ tokenEndpointAllowReusePvtKeyJwt }
+                                            name="tokenEndpointAllowReusePvtKeyJwt"
+                                            required={ false }
+                                            type="checkbox"
+                                            disabled={ isPublicClient }
```
Comment (pavinduLakshan) [inline features/admin.applications.v1/components/forms/inbound-oidc-form.tsx:1944]: Hope this is intentional.

File: modules/i18n/src/translations/en-US/portals/applications.ts (hunk 1 above)
Comment (mpmadhavig) [reply modules/i18n/src/translations/en-US/portals/applications.ts]: Updated the content with the previously approved org level config.

Screenshot 2024-07-02 at 07 05 48

The new UI will look like this. Changed the word `Token` to `Private Key JWT` as Token is too generic and can be confused with the Access Token.

Screenshot 2024-07-02 at 07 06 13

Comment (mpmadhavig) [reply modules/i18n/src/translations/en-US/portals/applications.ts]: @savindi7 @Achintha444 Please review. Will update the mail thread once this is approved.

**Comment by Achintha444** [comment]
Add changeset file

File: modules/i18n/src/translations/en-US/portals/applications.ts (hunk 1 above)
Comment (Achintha444) [reply modules/i18n/src/translations/en-US/portals/applications.ts]: +1 for the current text updated text.

**Review by pavinduLakshan — DISMISSED** [review]

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**